api/views/v1/generatedTasks.py

```diff
@@ -40,15 +41,18 @@ def post(self, request):
                 body: JSON.stringify({ name: 'New Project', projectID: '12345' }),
             });
         """
-        responseData, httpStatus = self.generateTasks(request.data)
+        responseData, httpStatus = self.generateTasks(
+            request.data, getAuthorizationToken(request)
+        )
```

Comment:
why do you need the AuthorizationToken parameter if you can call that method getAuthorizationToken directly in the method you're using it?

Reply:
`getAuthorizationToken(request)` requires a request object because the authorization (API) token can either be parsed from the request's authorization header (good for scripts that use the API) or from the cookies  (good for front end web). So this means, we would either have to pass the whole request to the object to get the authorization token or just pass the token like I do here. I asked chatGPT and it seemed better practice to pass this specific piece of info instead of the whole request.